### ISSUE TITLE
Unable to install contribution

### ISSUE THREAD

**Issue by apandurangi** [project-flogo/flogo-web#1193]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
When we try to install a new contribution to the WebUI fails with the following error in the server logs:
```
 Maximum call stack size exceeded {"stack":"RangeError: Maximum call stack size exceeded
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8670:23)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)
 at ContribInstallController.updateInstallState (/flogo-web/apps/server/main.js:8672:14)"}
```

**Expected behavior**
We should be able to  install new contributions to the WebUI

**Minimal reproduction of the problem with instructions**
1. Click on `Install Contribution` in the global header
2. Try to install `github.com/project-flogo/contrib/activity/jsexec` to the WebUI
3. Notice that the installation is failed with the above message in the server logs

* **Flogo version:** latest (master)

**Comment by tibmatt** [comment]
Whats the status of this? Not being able to add contributions is a bit of a blocker

**Comment by fcastill** [comment]
Just opened #1206 to fix it. It should only be affecting the master branch and not the latest released version (`v0.9.0`).

**Comment by tibmatt** [comment]
Thanks!
